### DISCUSSION TITLE
Make Ship Descriptions Great Again! (Aka it'd be nice to see the descriptions of ships we own)

### DISCUSSION THREAD

**Discussion by Fotiadis-110** [endless-sky/endless-sky#9317] [Lore]
TLDR: Ship model line in shipyard could do with tooltip "Click for Ship Description" and become clickable to read the description of ships owned by the player, instead of having to find a shipyard that sells it to discover a lot of hidden lore.
Buy and Sell Tooltips (outfitter). 'Click to Buy, Press I to install' 'Click To Sell, Press U to uninstall'
And maybe C should hotkey to 'swap' to Cargo mode, instead of trying to buy into outfit space. Are there other hotkeys I literally don't know about that need a hotkey description in an unintrusive format?

Been browsing the menus. Started modding the game a touch to make some of the empty fan made content either Viable, Interesting, Or in one case gave a region highly profitable 1000-15,000 population migration jobs so once you own a LARGE purpose built jump fleet, you can make money a non-issue despite the captured pirate warfleet costing you 80,000 credits per turn to ensure campaign victory/you can crush the unfetted without trying to Dominate the entire republic. And nerfing some of the weapons because clearly some Modders didn't think balance should be a thing. (Yes I notice the irony I've granted infinite money but want balanced warships)

However I'm already distracting myself.
My problem/request/desired fix is UI based, and having thought about it. I think I have a plan, if not the implementation.
The descriptions of Vanilla ships, Marauder, Korath, and other hostile factions, are never available to be browsed, despite almost every ship class in the game possessing interesting lore or at least a justification for its existance.
The only place you can find these descriptions is on ships FOR SALE in the shipyard.
But, with one tooltip and a clickable menu item in the Shipyard... we can fix that.
So when you scroll over the 'Your Ships' bar on the Right of the screen in both the Outfitter and Shipyard, hovering over various statistics and information will present tooltips to explain the purpose and relevance of the stat in question.

I suggest the ship stat: model: gain a tooltip "Click for ship description" and the name of the ship (and the word model, so modded ships lacking names can still be accessed presuming something isn't quite right with the hitbox) become clickable to open a simple mission type popup presenting the ships name, (image?) and it's description for folks like me who found the descriptions for the Korath warships fascinating but had to go file trawling to discover them, or spend time exploring fan made wiki's, which lack the modded content that also have their own descriptions!

Also. Is there any reason why the C key in the Outfitter will not let you buy items directly into cargo, and instead it tries to force you to buy into your outfit space? If its a engine limitation, Can I remap C to 'enter Cargo interface' so to buy up the 50 random rocks, I only have to push the button twice instead of getting error messages? Also the Uninstall and Install buttons always use the storage mechanic, and have no hotkey prompts on the screen or menus at all. Could we get a tooltip on the Buy and Sell buttons suggesting the alternative inventory management tools?

Also: Default Hotkeys. Afterburner on Shift. Hard Stop on Shift Backwards. Conflict. Hard stop stops you and afterburner pushes you away. I remapped afterburner to Q and moved Asteroid Scan to shift. Because Asteroid scanning doesn't force your ship to move when trying to stop. A simple Deconfliction shouldn't be that hard: Afterburner should be paused for ?half a second? when backwards is pressed (Permitting players to come to a halt and not immediately reverse thrust nor afterburn forwards again)
